documentation/CHANGELOG_EXAMPLE.md

```diff
@@ -0,0 +1,474 @@
+
+### Added
+
+* Added new endpoint to extend duration of an active session by @emil-cheung in https://github.com/camaraproject/QualityOnDemand/pull/216
+* Introduced of linting with Megalinter and Swagger Editor Validator by @RandyLevensalor, @maxl2287 and @ravindrapalaskar17 in https://github.com/camaraproject/QualityOnDemand/pull/206, https://github.com/camaraproject/QualityOnDemand/pull/207, https://github.com/camaraproject/QualityOnDemand/pull/212, and  https://github.com/camaraproject/QualityOnDemand/pull/215
+* Added global tags element  by @rartych in https://github.com/camaraproject/QualityOnDemand/pull/227
+* Added a new error example for DurationOutOfRangeForQoSProfile by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/259
+
+### Changed
+
+* Align event notification with CloudEvents spec by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/224
+* Moved "description" out of "allOf" declaration by @maxl2287 in https://github.com/camaraproject/QualityOnDemand/pull/205
+  * Note: this change shouldn't have an impact for API consumers but is relevant for implementations of the API.
+* Single IP addresses in Device model specified with standard formats instead of patterns by @jlurien in https://github.com/camaraproject/QualityOnDemand/pull/237
+* Moved "basePath" /qod/v0 to "url"-property and introduced "apiroot" in definition of server  @maxl2287 in https://github.com/camaraproject/QualityOnDemand/pull/252
+* Added statusInfo 'DELETE_REQUESTED' for qosStatus 'UNAVAILABLE' and clarified notification events in documentation by @hdamker in https://github.com/camaraproject/QualityOnDemand/pull/258
+
+### Fixed
+
+* NA
+  
+### Removed
+
+* NA
+
+## New Contributors
+* @ravindrapalaskar17 made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/215
+* @rartych made their first contribution in https://github.com/camaraproject/QualityOnDemand/pull/227
+
+**Full Changelog**: https://github.com/camaraproject/QualityOnDemand/compare/v0.9.0...v0.10.0
+
+# r2.4 - M4 release candidate
+
+## Release Notes
+
+* quality-on-demand v0.10.0-rc.2
+* Commonalities v0.2.0
+* ICM v0.1.0
+* Backward compatible: N/A
+
+**This release contains the second and final (M4) release-candidate for the v0.10.0 of the Quality-On-Demand (QoD) API**
+
+- API definition **with inline documentation**:
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.0-rc2/code/API_definitions/qod-api.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/QualityOnDemand/v0.10.0-rc2/code/API_definitions/qod-api.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/QualityOnDemand/blob/v0.10.0-rc2/code/API_definitions/qod-api.yaml)
+
+## Changes for quality-on-demand v0.10.0-rc.2
+
+Changes compared to r2.3
```

Comment:
```suggestion
### Changes for quality-on-demand v0.10.0-rc.2 compared to v0.10.0-rc.1

```

Reply:
It is stated in the beginning o comparisons are done - I would not add comparison version in the title in addition

Reply:
resolved